pymc/bart/utils.py

```diff
@@ -296,3 +299,68 @@ def plot_dependence(
 
     fig.text(-0.05, 0.5, Y_label, va="center", rotation="vertical", fontsize=15)
     return axes
+
+
+def variable_importance(idata, labels=None, figsize=None, samples=100, random_seed=None):
```

Comment:
Since its returning a plot, perhaps it should be called `plot_variable_importance` or similar?